### DISCUSSION TITLE
Unable to load the service index for nuget

### DISCUSSION THREAD

**Discussion by glabrie10** [uwrit/leaf#448] [Q&A] [answered]
When I try to compile the leaf API with no authentication I receive this error:

/usr/share/dotnet/sdk/3.1.413/NuGet.targets(128,5): error : Unable to load the service index for source https://api.nuget.org/v3/index.json

I have opened the IP address through a firewall and can ping https://api.nuget.org/v3/index.json but the get request is still failing.  Is there a config file I need to change?

**Comment by ndobb** [comment, marked as answer]
Hmmm no not that I can think of, @glabrie10. I do see [this seems to be a not uncommon issue](https://stackoverflow.com/questions/41185443/nuget-connection-attempt-failed-unable-to-load-the-service-index-for-source) though.

If you're not able to get it to work I'd suggest either using our pre-compiled builds under [Releases](https://github.com/uwrit/leaf/releases) or compiling the API from a build server/computer without potential firewall restrictions (e.g., a local laptop or other server) then copying the output to your intended deployment server.

**Reply by glabrie10** [reply]
I meant to come here and update you that I had used the precompiled version so this step did not matter.   I do appreciate the response.   Thank you very much!
